components/Homepage/Featured.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
You don't need this now but maybe later so you can keep it.

Also, it's better to import only what you need instead of the whole thing.

For example, this is better `import  {useEffect} from 'react';`